README.md

```diff
@@ -110,6 +110,11 @@ go install -ldflags "-X 'main.version=$(git describe --tags)' -X 'main.commit=$(
 
 On Windows, you can run the above commands with Git Bash, which comes with [Git for Windows](https://git-scm.com/download/win).
 
+### GitHub Actions installation
+
+To use GolangCI Lint as GitHub action in your repo please visit [GolangCI Lint Action](https://github.com/marketplace/actions/action-golangci-lint) on the Marketplace.
```

Comment:
please, transfer first `https://github.com/matoous/golangci-lint-action` to `golangci` org

Reply:
Hi! I tried to transfer the project under `golangci` but I get error message ` You don’t have the permission to create public repositories on golangci`. Can you give me permission to create a repo?

Reply:
sorry for the delay, I have created https://github.com/golangci/golangci-lint-action
@matoous you are the admin of the https://github.com/golangci/golangci-lint-action